src/test/java/mate/academy/service/impl/RoleServiceImplTest.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.service.impl;
+
+import mate.academy.model.Role;
+import mate.academy.service.RoleService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class RoleServiceImplTest {
+    private RoleService roleService;
+
+    @BeforeEach
+    void setUp() {
+        roleService = Mockito.mock(RoleService.class);
+    }
+
+    @Test
+    void save() {
+        Role role = new Role(Role.RoleName.USER);
+        role.setId(1L);
+        Mockito.when(roleService.save(Mockito.any())).thenReturn(role);
+        Role savedRole = roleService.save(role);
+        assertNotNull(savedRole.getId());
+    }
+
+    @Test
+    void getRoleByName() {
```

Comment:
```suggestion
    void getRoleByName_ok() {
```

lets allso add test for case where we getRoleByName with no existent role

Reply:
added test